src/cider/nrepl/middleware/debug.clj

```diff
@@ -438,12 +438,28 @@
             (eval form)
             (alter-meta! v assoc ::instrumented instrumented)))))))
 
+(def step-in-blacklist
+  "A set of names or regexes specifying namespaces that should not be stepped in
+  to. By default, this contains only \"clojure.core\"."
+  (atom #{"clojure.core"}))
+
+(defn blacklisted-for-step-in?
```

Comment:
It can be pretty frustrating to get a crash in the middle of a work session. Firstly because could lose work, secondly because CIDER takes almost a minute to jack-in again.